cmd/nsdns/update.go

```diff
@@ -24,18 +23,24 @@ func updateCommand() *cobra.Command {
 		Use:   "update",
 		Short: "update dns records",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			log.SetLevel(log.DebugLevel)
+
 			if domainName == "" {
 				return fmt.Errorf("Must provide a domain name to update DNS records.")
 			}
 
+			if ingressClass == "" {
+				return fmt.Errorf("Must provide an ingress class to select DNS records.")
+			}
+
 			nsApiKey := os.Getenv("NAMESILO_API_KEY")
 			if nsApiKey == "" {
 				return fmt.Errorf("Failed to find NAMESILO_API_KEY in environment. Cannot proceed.")
 			}
 
-			api := namesilo_api.NewNamesiloApi(nsApiKey)
+			api := namesilo_api.NewNamesiloApi(domainName, nsApiKey)
```

Comment:
Decided this kind of makes sense. If updating multiple domains, just have multiple instances.